dashboard/app/controllers/application_controller.rb

```diff
@@ -120,10 +120,11 @@ def milestone_response(options)
       timestamp: DateTime.now.to_milliseconds
     }
     script_level = options[:script_level]
+    level = options[:level]
```

Comment:
This method is just called from the activities and script_level controllers, and both of those default to something sane (or explicitly raise an exception).